README.md

```diff
@@ -65,6 +65,16 @@ Developers: Jason Huang, Soha Khan, Cindy Wang, Brandon Wong, Victor Yun, Mahad
 
 ## Run Locally
 
+To deploy your database schema run (one-time):
```

Comment:
Maybe it would be good to mention when making changes to the database, we want to reset database on heroku and run these again.

Reply:
done